lib/checkPullRequestBranch.js

```diff
@@ -23,28 +22,62 @@ const checkBranch = async (context) => {
    */
   const pullRequest = context.payload.pull_request;
   const branchName = pullRequest.head.ref;
-  const branchIsInvalid = DISSALLOWED_BRANCH_PREFIXES.some((prefix) => {
-    return branchName.startsWith(prefix);
-  });
 
-  if (branchIsInvalid) {
+  message = '';
+  invalid_branch_check = false;
+  
+
+  if(branchName === 'develop'){
+    
+    invalid_branch_check = true;
+    message = ', PRs made from develop branch are not allowed. Also branches '+
+              'whose name is prefixed with develop, release or test are not allowed.';
+  }
+
+  else if (branchName.startsWith('develop')) {
+
+    invalid_branch_check = true;
+    message = ', PRs made from a branch whose name is prefixed with develop ' +
+              'are not allowed. Also PRs made from develop branch or from a branch '+
+              'whose name is prefixed with release or test are not allowed.';
+  }
+
+  else if (branchName.startsWith('release')) {
+    
+    invalid_branch_check = true;
+    message = ', PRs made from a branch whose name is prefixed with release ' +
+              'are not allowed. Also PRs made from develop branch or from a branch '+
+              'whose name is prefixed with develop or test are not allowed.';
+  }
+
+  else if (branchName.startsWith('test')) {
+
+    invalid_branch_check = true;
+    message = ', PRs made from a branch whose name is prefixed with test ' +
+              'are not allowed. Also PRs made from develop branch or from a branch '+
+              'whose name is prefixed with develop or release are not allowed.';
+
+  }
```

Comment:
This can be simplified right? I would not mention the second part `Also PRs made from develop branch or from a branch whose name is prefixed with develop or test are not allowed.` an just iterate through the disallowed prefixes and if it is prefixed with it do `', PRs made from a branch whose name is prefixed with ${prefix} are not allowed.'`

Reply:
Done